contracts/cw1-whitelist-ng/src/msg.rs

```diff
@@ -1,147 +1,123 @@
 // This whole file except `AdminListResponse` shall be generated form contract traits and
 // instantiate signature
 
+use cosmwasm_std::{Binary, CustomMsg, Deps, DepsMut, Env, MessageInfo, Response};
 use schemars::JsonSchema;
-use serde::{Deserialize, Serialize};
+use serde::de::Error;
+use serde::{Deserialize, Deserializer, Serialize};
+use serde_cw_value::Value;
 
-use cosmwasm_std::{
-    to_binary, Binary, CosmosMsg, Deps, DepsMut, Empty, Env, MessageInfo, Response, StdError,
-};
+pub use crate::contract::msg::InstantiateMsg;
 
 use crate::error::ContractError;
-use crate::interfaces::*;
+pub use crate::interfaces::{cw1_msg, whitelist};
 use crate::state::Cw1WhitelistContract;
 
-#[derive(Serialize, Deserialize, JsonSchema)]
-pub struct InstantiateMsg {
-    pub admins: Vec<String>,
-    pub mutable: bool,
+#[derive(Serialize, Clone, PartialEq, JsonSchema, Debug)]
```

Comment:
This part was to be auto-generated, right?
Fine if you do that in a future PR (this is already a lot), but just want to clarify that this code (especially dispatch implementation) is not intended that people write in ng, right?

Reply:
Absolutely, as top level comment claims:

> "This whole file except `AdminListResponse` shall be generated

Generating those file is the next thing I am working on, should be done tomorrow. However, I want to close up some chapters - I came back to completely not working ng, now it works properly on blokchain.

Reply:
Missed this... just looking at diff.

Perfect 👌